app/products/calls/connection/connection.ts

```diff
@@ -203,8 +204,8 @@ export async function newConnection(
             }
         }
 
-        InCallManager.start({media: 'audio'});
-        InCallManager.stopProximitySensor();
+        InCallManager.start({media: 'video'});
```

Comment:
I have to ask, why `video`?

Reply:
That's the code for starting in "video" mode (speaker, no proximity sensor)